src/fs/v1/PublicFile.ts

```diff
@@ -43,7 +43,7 @@ export class PublicFile extends BaseFile implements HeaderFile {
   }
 
   protected async putWithKey(key: Maybe<string>) {
-    const { cid, size } = await basic.putFile(this.content, this.header.key)
+    const { cid, size } = await basic.putFileEncoded(this.content, this.header.key)
```

Comment:
wait this will add different file content to the `public` and `pretty` trees since one is encoded and one isn't :thinking: 

Not to self to fix this